journal-of-systematic-palaeontology.csl

```diff
@@ -171,6 +183,7 @@
       <key macro="author-count" names-min="3" names-use-first="3"/>
       <key macro="author" names-min="3" names-use-first="1"/>
       <key macro="year-date"/>
+     <key macro="month-date"/>
```

Comment:
(if you hear differently from the journal editors we'd of course be happy to change this)